doc/src/manual/faq.md

```diff
@@ -859,26 +859,36 @@ Modifying OpenBLAS settings or compiling Julia with a different BLAS library, eg
 
 ## Julia Releases
 
-### Do I want to use a release, beta, or nightly version of Julia?
-
-You may prefer the release version of Julia if you are looking for a stable code base. Releases
-generally occur every 6 months, giving you a stable platform for writing code.
-
-You may prefer the beta version of Julia if you don't mind being slightly behind the latest bugfixes
-and changes, but find the slightly faster rate of changes more appealing. Additionally, these
-binaries are tested before they are published to ensure they are fully functional.
+### Do I want to use the Stable, LTS, or nightly version of Julia?
+
+The Stable version of Julia is the latest released version of Julia,
+this is the version most people will want to run.
+It has the latest features, including improved performance.
+The Stable version of Julia will always be versioned according to SemVer as v1.x.y.
+A new minor release of Julia corresponding to a new Stable version is made approximately every 4 months after a few weeks of testing as a release candidate.
+Unlike the LTS version the a Stable version will not normally recieve bugfixes after another Stable version of Julia has been released.
+However, upgrading to the next Stable release will always be possible as each release of Julia v1.x
+will continue to run code written for earlier versions.
+
+You may prefer the LTS (Long Term Support) version of Julia if you are looking for a very stable code base.
+The LTS version of Julia will always be versioned according to SemVer as v1.0.x.
+It will continue to recieve bugfixes for the lifetime of Julia version 1, but will not recieve any new features.
+As a package developer, you may prefer to develop for the LTS version, to maximize the number of users who can use your package.
+Code written for v1.0 will continue to work for all future LTS and Stable versions.
+In general even if targetting the LTS, one can develop and run code in the latest Stable version,
+to take advantage of the improved performance; so long as one is careful not to use any of the new features (such as added library functions).
+Patch releases to the LTS version generally occur approximately at the same time as releases to the Stable version.
 
 You may prefer the nightly version of Julia if you want to take advantage of the latest updates
 to the language, and don't mind if the version available today occasionally doesn't actually work.
+As the name implies, releases to the nightly version are made every night.
+In general nightly released are fairly safe to use.
+However, they may have regressions and or issues that have not yet been caught.
+You may wish to test against the nightly version to ensure that such regressions that affect your use case are caught before a release is made.
```

Comment:
I think this sounds a bit too alarming. There is a decently long RC process where there is time to test for regressions.

Reply:
again it says _may_.
We have had breaking changed caught (and recently) thanks to people doing this,
I don't see any reason for them to stop.